percona-server-mongodb.36/ps-entry.sh

```diff
@@ -136,6 +136,41 @@ _mongod_hack_ensure_arg_val() {
 	mongodHackedArgs+=( "$ensureArg" "$ensureVal" )
 }
 
```

Comment:
please add comment before each function for which version  of psmdb it is needed

Reply:
Added!